enhancements/machine-config/manage-boot-images.md

```diff
@@ -115,23 +119,37 @@ Any form factor using the MCO and `MachineSets` will be impacted by this proposa
 
 ##### Supported platforms
 
-The initial release(phase 0) will support GCP. In future releases, we will add in support for remaining platforms as we gain confidence in the functionality and understand the specific needs of those platforms. For platforms that cannot be supported, we aim to atleast provide documentation to perform the boot image updates manually. Here is an exhaustive list of all the platforms:
+The initial release(phase 0) will support GCP. In future releases, we will add in support for remaining platforms as we gain confidence in the functionality and understand the specific needs of those platforms. For platforms that cannot be supported, we aim to at least provide documentation to perform the boot image updates manually. Here is an exhaustive list of all the platforms:
 
-- gcp
 - aws
 - azure
-- alibabacloud
+- baremetal
+- gcp
+- ibmcloud
 - nutanix
-- powervs
 - openstack
+- powervs
 - vsphere
-- baremetal
-- libvirt
-- ovirt
-- ibmcloud
 
 This work will be tracked in [MCO-793](https://issues.redhat.com/browse/MCO-793).
 
+##### Projected timeline
+
+This is a tentative timeline, subject to change (GA = General Availability, TP = Tech Preview, EF = Default-on, Skew Enforced).
```

Comment:
We discussed this briefly, but it's probably good to de-couple default-on and skew-enforced. There's a few benefits:
1. as all users of one platform gets on the auto update path, we get a chance to iron out any breakage/edge cases
2. we can simplify implementation a bit and wait until all platforms have it on before we turn on enforcement, giving us some extra time to refine that workflow